pip-requirements.txt

```diff
@@ -0,0 +1,8 @@
+ckantoolkit==0.0.3
+pika>=1.1.0
+pyOpenSSL==18.0.0
+redis
+requests>=2.11.1
+six>=1.12.0
```

Comment:
do we need all of this for archiver? ie messytables?

Reply:
Probably not. Where did this file come from?

Reply:
xloader setup